README.md

```diff
@@ -52,8 +52,8 @@ Rscript -e 'devtools::install()'
 Given the street network of the city of Bucharest (data source: [OpenStreetMap][osm])
```

Comment:
Should we add a note that `CRiSpData` must be installed: `Rscript -e 'devtools::install_github("CityRiverSpaces/CRiSpData")'`? or the dev installation code block, above, should include `dependencies=TRUE`